# Analysis of Results

As a full proteinswap calculation involves over 800 million Monte Carlo moves, we have pre-prepared the output for you to analyse. This is in the directory `example_output`.

As proteinswap runs, it stores all of the free energy differences and gradients necessary to calculate free energies into a file in this directory called `freenrgs.s3`. This is a binary file that can be analysed using the program `analyse_freenrg` which comes with Sire.

In [1]:
! $SIRE/bin/analyse_freenrg -i example_output/freenrgs.s3

Starting /Users/chris/sire.app/bin/analyse_freenrg: number of threads equals 4

# Writing all output to stdout
# Analysing free energies contained in file(s) "example_output/freenrgs.s3"
# Processing object Bennetts( nWindows() == 18, nIterations() == 1001, nSamples() == 1600000000 )
# Averaging over iterations 400 to 1000
# Processing object FEP( nWindows() == 18, nIterations() == 1001, nSamples() == 1600000000 )
# Averaging over iterations 400 to 1000
# Processing object TI( nLambdaValues() == 16, nIterations() == 1001, nSamples() == 1600000000 )
# Averaging over iterations 400 to 1000
# Convergence
# Iteration 
# Bennetts # FEP # TI 
1 35.10340379784204 204.3225194227021 53.22454647526262 
2 19.88569623617181 17.41760310283212 18.120980861596678 
3 18.997991199345073 16.90859085583587 15.77830304672405 
4 9.494151976985654 8.410518311461654 8.337785559036462 
5 18.373793251216085 14.8470443988397 14.278374812575358 
6 17.339609335887705 14.535224367085833 15.031994396985436 
7 16.04

655 9.680271996835579 7.573452044108367 6.546151022728847 
656 13.13313564188011 9.786711509463505 8.484084745645703 
657 8.85876829346692 5.926144639417455 4.533158580144399 
658 6.612372806870776 5.12899498166014 3.0410496188538234 
659 13.991497371219483 10.19072321557719 9.335757544681213 
660 10.912142050113113 8.070874873632441 8.092476534748046 
661 5.195555632960634 4.201372709598347 5.697181231031947 
662 8.210088677774047 6.5489375420355795 6.961062451941219 
663 8.301104972476095 6.669633761821724 7.544298763098615 
664 9.47214731013274 6.57799596507635 5.317909163273612 
665 13.486655937813259 9.498761441436411 9.339547064038092 
666 12.299783546132447 9.276967566710482 10.38238837099707 
667 10.303183972313736 8.719194038169427 10.373697491189022 
668 12.261185948678031 10.575343029488502 14.185131377921783 
669 15.493681619721537 11.349973048824035 12.168399343463866 
670 17.12903539850219 12.089926625102278 12.544335144201659 
671 12.953211736894128 9.683185173626706 10.


Plot of free energy versus iteration
   │                                                                            
   │                                                      .                     
   ┼+25.6674     .           ..                              .                  
   │            ..               .       . . .       ..  ...          .         
   │           ...      .    ..    .     .  ..  .   ..   ..  .                  
   │        .  .....   ..  . ..  .   .   ... . .  . ...  ... .   .   ...        
   │       .. .....   ... .. ... . .    ....... . . ...........  .   ....  .    
   │       . ....................... .. .......... ..... .........   ....  .    
   │       ........................... ........................... . .......    
   │       ......................... ............................... .......    
   │      ....... ...........................................................   
   │       ...... .................................. ..................

0.37000000000000016  2.5643024069132343  2.7668776009371925  2.361727212889276
0.38000000000000017  2.43423689269767  2.6359690886172973  2.2325046967780424
0.3900000000000002  2.321590390991056  2.5222001230234445  2.1209806589586675
0.4000000000000002  2.2276112898913976  2.4269579001479022  2.028264679634893
0.4100000000000002  2.1534016614183575  2.351487021963003  1.955316300873712
0.4200000000000002  2.099917220931717  2.2973511539822122  1.9024832878812217
0.4300000000000002  2.0679674269789974  2.2711685338271046  1.8647663201308902
0.4400000000000002  2.058215605306568  2.2673771466552086  1.8490540639579276
0.45000000000000023  2.0711790140227198  2.2863297628145665  1.856028265230873
0.46000000000000024  2.1072288003195654  2.328232109203785  1.8862254914353458
0.47000000000000025  2.1665898319545667  2.3931489229775513  1.940030740931582
0.48000000000000026  2.2493404181451213  2.48100967308319  2.0176711632070528
0.49000000000000027  2.355411963935131  2.5916138066729992  

The output is the same and is interpreted in the same way as for ligandswap. Here, we see that we have a smooth PMF with good convergence, and with all free energy methods in good agreement.

```
PMF Plot of free energy versus lambda
   │                                                                            
   ┼+11.1294                                                   ○○○○             
   │                                                        ○○○○   ○○○          
   │                                                      ○○          ○○        
   │                                                    ○○             ○○       
   │                                                   ○                 ○      
   │                                                 ○○                   ○     
   │                                                ○                     ○     
   │                                              ○○                       ○○   
   │          ○○○                                ○○                             
   │      ○○○○   ○○○○○                         ○○                               
   │     ○○          ○○○                      ○○                                
   │   ○○              ○○○                  ○○                                  
   │  ○○                  ○○○             ○○                                    
   │ ○                       ○○○       ○○○                                      
   │ ○                          ○○○○○○○                                         
   │○                                                                           
   ┼+0.24                                                                       
───○┼───────────────────────────────────────────────────────────────────────┼───
   │ +0.0051                                                              +1    

# Free energies 
# Bennetts = 6.578266930672087 +/- 0.014673283219845916 kcal mol-1#
# FEP = 6.4112676361336005 +/- 7.487409492160177 kcal mol-1#
# TI = 6.965892869264752 +/- 0.44346741599550965 kcal mol-1 (quadrature = 6.872805678979021 kcal mol-1)#
```

This shows that proteinswap predicts (correctly) that the R292K mutant is a weaker binder to oseltamivir, with a free energy difference of between 6.5 and 7.0 kcal mol-1.